css/index.css

```diff
@@ -136,11 +205,154 @@ footer nav {
     display: flex;
     justify-content: space-between;
     align-items: center;
-    padding: 20px 2%;
-    font-size: 14px;
+    padding: 2rem 2%;
+    font-size: 1.4rem;
 }
 
 footer nav a {
     color: white;
     text-decoration: none;
+}
+
+/* TABLET 768PX */
+@media (max-width: 768px) {
+    .container {
+        width: 100%;
+    }
+
+    /* HEADER */
+    header {
+        display: flex;
+        flex-wrap: wrap;
+    }
+
+    header nav {
+        margin: 1rem 8% 2rem 5%;
+        font-size: 1.5rem;
+    }
+
+    header nav a {
+        margin-right: 18%;
+        margin-left: 3%;
+    }
+
+    img {
+        width: 99%;
+        margin: 2rem 0 2rem 0;
+    }
+
+    header .logo {
+        width: 20%;
+        display: block;
+        margin: 2rem auto;
+        justify-content: center;
+        align-items: center;   
+    }
+
+    /* MIDDLE CONTENT */
+    .middle-content .boxes .box:nth-child(1) {
+        background: teal;
+    }
+
+    .middle-content .boxes .box:nth-child(3) {
+        background: gold;
+    }
+
+    .middle-content .boxes .box:nth-child(5) {
+        background: cadetblue;
+    }
+
+    .middle-content .boxes .box:nth-chil(7) {
+        background:coral;
+    }
+
+    .middle-content .boxes .box:nth-child(9) {
+        background: crimson;
+    }
+}
+
+/* PHONE */
```

Comment:
If you take a closer look at the design files, box 10 is not displayed! It's some sneaky stuff and you can always overwrite it in here.